microservice/frontend/catalog/debian/data/etc/nginx/sites-available/shop.microservice.io

```diff
@@ -1,3 +1,17 @@
+
+upstream _backend_old {
```

Comment:
instead of _backend_old, _backend_new, I suggest _site_A, _site_B. Easier to understand and explain during training